extensions/amp-form/0.1/amp-form.js

```diff
@@ -234,9 +234,15 @@ export class AmpForm {
         continue;
       }
 
-      if (input.type == 'checkbox' || input.type == 'radio') {
-        data[input.name] = input.checked ? input.value : '';
-      } else {
+
+      if (input.type != 'checkbox' && input.type != 'radio') {
+        data[input.name] = input.value;
+      } else if (input.type == 'checkbox' && input.checked) {
+        if (data[input.name] === undefined) {
+          data[input.name] = [];
+        }
+        data[input.name].push(input.value);
```

Comment:
Since we are doing this now, can we do it for all inputs regardless of type? Different inputs with the same name is kind of valid (although HTML5 doesn't officially allow it anymore) so when encountering that, sending an array of value instead of last one overriding it all would be more backward compatible (that being said, I don't know what `FormData` does in that case, might be worth a quick test and do whatever `FormData` does)